include/seqan3/alignment/matrix/detail/two_dimensional_matrix.hpp

```diff
@@ -117,7 +117,7 @@ class two_dimensional_matrix
      */
     template <std::ranges::forward_range entries_t>
```

Comment:
documentation of this template argument is missing
also `\param` is missing the `[in]` specifier

Reply:
:sob: